File: Semestrul 4/MPP2/Client-Server_2Limbaje/Problema8SC/Problema8SC/Problema8Networking/src/main/java/problema8/network/dto/UserDTO.java
package problema8.network.dto;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

public class UserDTO implements Serializable {

    @SerializedName("Username")
    private String Username;

    @SerializedName("Password")
    private String Password;

    public UserDTO(String username, String password) {
        this.Username = username;
        this.Password = password;
    }

    public String getUsername() {
        return Username;
    }

    public void setUsername(String username) {
        this.Username = username;
    }

    public String getPassword() {
        return Password;
    }

    @Override
    public String toString() {
        return "UserDTO[" + Username + " " + Password + "]";
    }
}

File: Semestrul 4/MPP2/Client-Server_2Limbaje/Problema8SC_CSharp/Networking/dto/UserDTO.cs
namespace Networking.dto;

[Serializable]
public class UserDTO
{
    public string Username { get; set; }
    public string Password { get; set; }

    public UserDTO(string username, string password)
    {
        Username = username;
        Password = password;
    }

    public override string ToString()
    {
        return $"UserDTO[{Username} {Password}]";
    }
}
